### ISSUE TITLE
Add in all needed interface variable strings to the keyboard constants

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Android#214]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

Post a recent discussion in a Scribe dev sync, we decided that all variables that are required for functioning keyboards should be hard coded into the application rather than localized via Weblate. Based on this, we need the following strings hard coded into each keyboard file, with the examples for Spanish being below:

- currencySymbol = "$"
- spaceBar = "espacio"
- language = "Español"
- invalidCommandMsg = "No en Wikidata"
- translateKeyLbl = "Traducir"
- translatePlaceholder = "Ingrese una palabra"
- translatePrompt = commandPromptSpacing + "es -› \(getControllerLanguageAbbr()): "
- translatePromptAndCursor = translatePrompt + commandCursor
- translatePromptAndPlaceholder = translatePromptAndCursor + " " + translatePlaceholder
- + the equivalents for this for conjugate and plural as well

See the [Scribe-iOS interface variables for Spanish](https://github.com/scribe-org/Scribe-iOS/blob/2d60078480cdae56e6cc2eac0375fcb1c62e0e09/Keyboards/LanguageKeyboards/Spanish/ESInterfaceVariables.swift#L258) for further information :)

### Contribution

Happy to discuss the needs here and review when a PR is up 😊

**Comment by Saifuddin53** [comment]
Great then I would like to get started with this issue @angrezichatterbox

**Comment by andrewtavis** [comment]
Thanks for you vested interest in working with us, @Saifuddin53! Please let us know if you have any questions 😊

**Comment by Saifuddin53** [comment]
Starting this one @andrewtavis but before that it would be great if you could provide me with some more context about how we are going to use these constant variables as I am not able to use the ios version of this app I couldn't get too much from the link you provided 🙂

**Comment by andrewtavis** [comment]
Those variables are basically going to be used within the interface of the keyboard, @Saifuddin53 :) So some explanations of the above are:

```
  currencySymbol = // base currency symbol for the keyboard
  currencySymbolAlternates = // which alternate characters should the above key have when we long press it
  spaceBar = // what's space called in the language
  language = // what's the name of the language
  invalidCommandMsg = // what's "Not in Wikidata" in the language (we'll show this when we don't have data)
  baseAutosuggestions = // what are the base autosuggestions that are shown to the user if we have no others
  numericAutosuggestions = // which autosuggestions are shown after numbers
  verbsAfterPronounsArray = // which verbs are shown after the user types a pronoun
  pronounAutosuggestionTenses = // which tenses of verbs should be shown after the user types a pronoun - so 2nd person singular if they type "you"

  translateKeyLbl = // what's "Translate" in the language - the label of the translate command when they select it
  translatePlaceholder = // the placeholder for the translate command - "enter a word"
  translatePrompt = // the prompt for the translate command, so the user will see the ISO-2 code and an arrow to the target language
  translatePromptAndCursor = // combination of the above prompt and the cursor for checks to prevent deletion
  translatePromptAndPlaceholder = // combination with the placeholder text
  translatePromptAndColorPlaceholder = // add mutability so we can make the placeholder grey, which is done in the next line

  ... The same for conjugate and plural :)
```

Basically we'll need to move all of the above variables over to this app, @Saifuddin53 😊

CC @angrezichatterbox and @Linfye as I think that the above will be helpful to understand what's going on in the keyboard UI 🚀

**Comment by andrewtavis** [comment]
Closed by #255 :) Thanks so much for your support, @Saifuddin53! 🚀 Let us know when you find another issue of interest 😊
